build/sbt

```diff
@@ -36,7 +36,7 @@ fi
 declare -r noshare_opts="-Dsbt.global.base=project/.sbtboot -Dsbt.boot.directory=project/.boot -Dsbt.ivy.home=project/.ivy"
 declare -r sbt_opts_file=".sbtopts"
 declare -r etc_sbt_opts_file="/etc/sbt/sbtopts"
-declare -r default_sbt_opts="-Xss64m"
+declare -r default_sbt_opts="-Xss64m -XX:+IgnoreUnrecognizedVMOptions -XX:+UnlockDiagnosticVMOptions -XX:GCLockerRetryAllocationCount=100"
```

Comment:
add `-XX:+IgnoreUnrecognizedVMOptions` for safety, since `-XX:GCLockerRetryAllocationCount` is going to be removed in JDK 25, see [JDK-8357684](https://bugs.openjdk.org/browse/JDK-8357684)

Reply:
I've learned about this options, but it's just a solution that tries to mitigate the problem as much as possible, right?

Reply:
yes, exactly

Reply:
I remain neutral on this matter and would rather see a more thorough solution.

Reply:
I'm afraid the GCLocker issue can not be thoroughly solved until upgrading to Java 25 (suppose Spark will only adopt Java LTS). Trino used to add such Java opts to its default `jvm.config`. https://github.com/trinodb/trino/pull/19026

Reply:
Could we synchronously make the same fix on the Maven side? We need to ensure the consistency of these options as much as possible.